cellprofiler/modules/removeobjects.py

```diff
@@ -42,6 +45,21 @@ def visible_settings(self):
         ]
 
     def run(self, workspace):
-        self.function = skimage.morphology.remove_small_objects
+        self.function = lambda labels, diameter: remove_objects(labels, diameter)
 
         super(RemoveObjects, self).run(workspace)
+
+
+def remove_objects(labels, diameter):
```

Comment:
I think the parameter makes sense for scikit-image. I think the documentation could be improved.